src/components/badge/examples/badge-number.tsx

```diff
@@ -1,8 +1,16 @@
 import { Component, h } from '@stencil/core';
 
+/**
+ * Number badges
+ *
+ * Numeric labels larger than 999 will get both rounded and abbreviated.
+ * For example, if the label is `1090` the badge will display `1.1K`.
+ * Abbreviation units used are `K` (Kilo) that stands for Thousands,
```

Comment:
```suggestion
 *
 * Abbreviations are indicated by `K` (Kilo) that stands for Thousands,
```

Reply:
Not sure how picky we should be or if this is just confusing... but if we are using SI units, uppercase `K` actually stands for kelvin and measures degrees :smile: Lowercase `k` stands for `kilo`
https://en.wikipedia.org/wiki/International_System_of_Units

Reply:
Wow...! I had no clue. But doesn't `MM` actually stand for millions? So I don't know neither. Maybe it's just a UI matter here. What do you think @LucyChyzhova ?

I myself prefer lowercase for everything to be honest. 
Uppercase is kind of both: 
- aggressive & ugly; 
- and also makes it harder to see where the number ends and the letter starts (for my dyslexic brain at least). 
 
Maybe we should convert all to lowercase instead?

Reply:
I'm pretty sure nobody except you and they guy who write the Wiki knows this 😄 Maybe we should just do whatever we want

Reply:
Haha, most people that have studied math or physics probably know it :smile:

Reply:
I think it's should be uppercase M. agree with Johan, this convention comes from the math lessons 🤓

Reply:
I'm intrigued :eyes: :smile:

Reply:
It's OK. 😄 
I think in the contexts where badges are used, nobody would interpret their values as Kelvin. Even those who know the difference. I mean you normally won't use a badge to visualize the temperature or things like that. They're always counters.
<img width="659" alt="image" src="https://user-images.githubusercontent.com/35954987/195590016-84195bc1-f861-4970-8e19-bdb6ad48f7a8.png">

But this is an unimportant thing really, the lowercase works too.

I also like the idea of adding a space. Tried it in the image above ☝️

Reply:
It's not that people might misunderstand. It's that it's very annoying and makes us look ignorant.

Or to put it the Günther Steiner way: https://youtu.be/W5WVVls74mI?t=25 😂